apps/teams-test-app/src/public/auth_start.html

```diff
@@ -10,8 +10,13 @@ <h2 style="font-size: 40px;">Starting auth</h2>
       const mockOAuth = params.get('mockOAuth');
       const authId = params.get('authId');
       const method = params.get('oauthRedirectMethod');
+      const redirectUrl = params.get('hostRedirectUrl')
       const hostName = params.get('hostName');
-      const state = `{"authId":"${authId}","method":"${method}", "hostName":"${hostName}"}`;
+      const state = `{"authId":"${authId}","method":"${method}","hostName":"${hostName}"}`;
+
+      if (redirectUrl.trim() !== '') {
```

Comment:
>if (redirectUrl.trim() !== '') [](http://example.com/codeflow?start=6&length=30)

`URLSearchParams` is documented as returning `null` if a search parameter isn't present, should we be using a `null` check here instead? In the case where it's not null but completely whitespace, I would wonder whether we should do nothing special in that case (e.g. don't `trim()`) because that case would be a bug and we'd want to it fail? The code should never be just giving whitespace.

Reply:
That makes sense, I'll just do a null check instead and let whitespace cases go through